linux-tkg/README.md

```diff
@@ -0,0 +1,70 @@
+**Due to intel_pstate poor performances as of late, I have decided to set it to passive mode to make use of the acpi_cpufreq governors passthrough, keeping full support for turbo frequencies.**
+
+### PDS, MuQSS and BMQ are not yet available options for 5.9 RC
+### MuQSS is not an available option for 5.8.y yet
```

Comment:
Just a small change but this probably should get changed to `MuQSS is not an available option for 5.8+ yet` or something like that